projects/batfish/src/test/java/org/batfish/bdp/BdpDataPlanePluginTest.java

```diff
@@ -149,9 +149,19 @@ public void testApplySourceNatInvalidAclThrows() {
   public void testIbgpRejectOwnAs() throws IOException {
     String testrigName = "ibgp-reject-own-as";
     String[] configurationNames = new String[] {"r1", "r2a", "r2b"};
+    String[] bgpTableNames = new String[] {};
+    String[] hostFilenames = new String[] {};
+    String[] iptablesFilenames = new String[] {};
+    String[] routingTableNames = new String[] {};
```

Comment:
suggest general inlining of these variables throughout PR, (if you feel like it). If only used once, may make code more readable.